core/account.js

```diff
@@ -87,6 +87,15 @@ Account.prototype.tx = function tx(to, value, data='0x00') {
     return '0x' + tx.serialize().toString('hex');
 };
 
+/**
+ * Get address of account as hex string.
+ *
+ * @returns {string} Address as hex string.
+ */
+Account.prototype.getStrAddress = function getStrAddress() {
```

Comment:
It's probably should be called 'getAddressHex' or 'getHexAddress'. Buffer can be transformed to string in a numerous ways, so calling it 'str' is not the best way of describing what this function does.